.github/workflows/snapshot-build.yml

```diff
@@ -11,21 +11,37 @@ on:
     tags:
       - 'snapshot-*'
 
+# Incremental compilation here isn't helpful
+env:
+  CARGO_INCREMENTAL: 0
+
 jobs:
-  container-images:
-    # Only run for releases
-    if: github.event_name == 'push' && github.ref_type == 'tag'
+  container-linux:
     runs-on: ubuntu-20.04
     permissions:
       contents: write
       packages: write
     strategy:
       matrix:
         image:
-          - node
           - farmer
+          - node
+        platform:
+          - arch: linux/amd64
+            profile: production
+            suffix: ubuntu-x86_64-${{ github.ref_name }}
+#          - arch: linux/arm64
```

Comment:
remove if not required ?

Reply:
I was to re-enable it soon, decided to leave for now